static/js/directives/ui/file-upload-box.js

```diff
@@ -31,50 +32,50 @@ angular.module('quay').directive('fileUploadBox', function () {
       $scope.state = 'clear';
       $scope.selectedFiles = [];
 
-      var conductUpload = function(file, url, fileId, mimeType, progressCb, doneCb) {
+      var conductUpload = function (file, url, fileId, mimeType, progressCb, doneCb) {
         var request = new XMLHttpRequest();
         request.open('PUT', url, true);
         request.setRequestHeader('Content-Type', mimeType);
-        request.onprogress = function(e) {
-          $scope.$apply(function() {
+        request.onprogress = function (e) {
+          $scope.$apply(function () {
             if (e.lengthComputable) { progressCb((e.loaded / e.total) * 100); }
           });
         };
 
-        request.onerror = function() {
-          $scope.$apply(function() { doneCb(false, 'Error when uploading'); });
+        request.onerror = function () {
```

Comment:
@ibazulic 
It looks like an empty space was placed between the word `function` and the parenthesis all throughout this file. It might be your IDE or editor's autoformatting.